src/client/styles/base.scss

```diff
@@ -2,6 +2,7 @@
 @import "nhsuk-frontend/dist/scss/nhsuk";
 
 // components (eg. site search, navigation)
+@import "components/form-component";
```

Comment:
This file name doesn't describe the object defined within it very clearly. Good practice is to name the file after the block (based on BEM) that is defined within that file.